### PR TITLE
Improve the sentence validator for yue

### PR DESCRIPTION
**Pull request by laubonghaudoi**
Changes include:
1. More comments on the rules
2. Add restrictions of no Mandarin characters

### CODE REVIEW

File: server/lib/validation/languages/yue.js [hunk 1]
```diff
@@ -25,8 +32,16 @@ const INVALIDATIONS = [{
...
+  // 吧唄吶
   regex: /[\u5427\u5504\u5436](\s|\u3002|\u002E|\uFF0C|\u002C|$)/,
-  error: '句子唔可以有官話語氣詞（例如吧、唄、吶）',
+  error: '句子唔可以有官話語氣詞（吧唄吶）',
+},
+{
+  // No Mandarin characters
+  // 這哪您們唄咱啥甭
+  regex: /[\u9019\u54EA\u60A8\u5011\u54B1\u5565\u752D]/,
+  error: '句子唔可以有官話詞（這哪您們咱啥甭）',
```
Comment (MichaelKohler) [inline server/lib/validation/languages/yue.js:44]: Are these the most common characters?

**Review by MichaelKohler — APPROVED** [review]
Looks good to me, thanks!

File: server/lib/validation/languages/yue.js (hunk 1 above)
Comment (laubonghaudoi) [reply server/lib/validation/languages/yue.js:44]: Yes, this is not an exhaustive list of Mandarin-unique characters, but the most common ones.

Comment (MichaelKohler) [reply server/lib/validation/languages/yue.js:44]: Ok, great. Thanks!

**Pull request merged**